apps/zotonic_core/src/zotonic_core.erl

```diff
@@ -76,16 +75,14 @@ set_configs() ->
     application:set_env(setup, log_dir, z_config:get(log_dir)),
     application:set_env(setup, data_dir, z_config:get(data_dir)),
     % Store filezcache data in the cache_dir.
-    FileZCache = filename:join([ z_config:get(cache_dir), "filezcache", atom_to_list(node()) ]),
-    application:set_env(filezcache, data_dir, filename:join([ FileZCache, "data" ])),
-    application:set_env(filezcache, journal_dir, filename:join([ FileZCache, "journal" ])).
+    FileZCache = filename:join([z_config:get(cache_dir), "filezcache", atom_to_list(node())]),
+    application:set_env(filezcache, data_dir, filename:join([FileZCache, "data"])),
+    application:set_env(filezcache, journal_dir, filename:join([FileZCache, "journal"])).
 
 maybe_start_logstasher() ->
-    IsLogstasherNeeded = lists:any(
-        fun(#{ module := Module }) ->
-            Module =:= logstasher_h
-        end,
-        logger:get_handler_config()),
+    IsLogstasherNeeded =
+        lists:any(fun(#{ module := Module }) -> Module =:= logstasher_h end,
+                  logger:get_handler_config()),
```

Comment:
Personally I find this less readable than the non-reformatted version, also because the arguments are not a tab stop anymore.

Reply:
You can add `-format #{inline_simple_funs => false}.` to this module or the equivalent option to the `rebar.config` file if you want it for all modules.